tests/core/fs/test_fs_offline_restart_and_rwfile.py

```diff
@@ -43,9 +45,9 @@ async def atomic_read(self, size, offset):
             content=st.binary(max_size=PLAYGROUND_SIZE),
         )
         async def atomic_write(self, offset, content):
-            await self.workspace.write_bytes(
-                "/foo.txt", data=content, offset=offset, truncate=False
-            )
+            async with await self.workspace.open_file("/foo.txt", "rb+") as f:
```

Comment:
shouldn't it be `wb` mode there ?

Reply:
No because we do not want to create the file if it doesn't exist. The previous version of `write_bytes` used to match this behavior but it was a mistake as `write_bytes` is expected to create a file if it doesn't exist (as in the `pathlib.Path.write_bytes` definition)